packages/fast-form-generator-react/src/utilities/form-input.style.ts

```diff
@@ -165,7 +165,7 @@ export function applyHeaderStyle(): ICSSRules<{}> {
 
 export function applyGlobalStyle(): ICSSRules<{}> {
     return {
-        "body > li[draggable=\"true\"]": {
+        "body > div > li[draggable=\"true\"]": {
```

Comment:
Can't find where the markup changes are, but `<li>` is technically not a valid list unless contained within a `<ul>`.

Reply:
it's still contained within a ul when it's not being dragged.

Reply:
Gotcha - makes sense